tasks/grunt-stylelint.js

```diff
@@ -1,15 +1,20 @@
 /*!
  * Run CSS files through stylelint and complain
  */
-var chalk = require( 'chalk' );
+var chalk = require( 'chalk' ),
+	stripAnsi = require( 'strip-ansi' );
```

Comment:
This is what upstream uses; I'll file an Issue for them to expose this feature in the API and not just the CLI so we can remove this hack.

Reply:
Filed as stylelint/stylelint/issues/4603.

Reply:
Fixed by using FORCE_COLOR instead.